Services/Cache.cs

```diff
@@ -18,7 +19,7 @@ public interface ICache
 
         Task<CacheValue> SetCacheAsync(CacheValue cache);
 
-        Task RebuildCacheAsync(bool force = false);
+        Task<bool> RebuildCacheAsync(bool force = false);
```

Comment:
nit: name the function something bool related if it's returning a bool.

Reply:
Fixed